CMakeLists.txt

```diff
@@ -522,8 +522,8 @@ else()
     add_executable(xmr-stak ${SRCFILES_CPP})
 endif()
 
-set(EXECUTABLE_OUTPUT_PATH "bin")
-set(LIBRARY_OUTPUT_PATH "bin")
+set(EXECUTABLE_OUTPUT_PATH "bin" CACHE STRING "Path to place executables")
```

Comment:
please add extent the description that this path is relative to `${CMAKE_INSTALL_PREFIX}`

Reply:
I forced pushed the updated working